test/user.spec.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest');
+const app = require('../app');
+
+const userCreate = {
```

Comment:
These jsons' names are a little bit confusing.
Also I would move them to another file